tests/app/main/views/test_dashboard.py

```diff
@@ -845,15 +844,16 @@ def test_usage_page_no_sms_spend(
     )
 
     annual_usage = page.find_all('div', {'class': 'govuk-grid-column-one-half'})
-    sms_column = normalize_spaces(annual_usage[0].text + annual_usage[1].text)
+    sms_column = normalize_spaces(annual_usage[0].text)
     assert 'Text messages' in sms_column
+    assert '1,000 sent' in sms_column
     assert '250,000 free allowance' in sms_column
     assert '249,000 free allowance remaining' in sms_column
     assert '$0.00 spent' not in sms_column
     assert 'pence per message' not in sms_column
 
 
-@freeze_time("2012-03-31 12:12:12")
+@ freeze_time("2012-03-31 12:12:12")
```

Comment:
Minor, but these shouldn't have a space after the `@`

Reply:
Hmm not sure why this injected a space here, fixed